### PR TITLE
[FR] Review of chapters 0, 2 & 5 + add chapters 6, 7, 8 & event

### PR DESCRIPTION
**Pull request by lbourdois**
Hi,

As mentioned in the title : 
- review of chapters 0, 2 & 5 (I think these chapters are now finished)
- addition of chapters 6, 7, 8 & event (these 4 chapters will be proofread later. It is easier to do it once the chapters are visible on the site because the errors are more apparent than in the markdown preview)

### CODE REVIEW

File: chapters/fr/chapter2/8.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@
 			text: " Tout d'abord, le modèle, qui traite le texte et renvoie des prédictions brutes. Puis le <i>tokenizer</i> donne un sens à ces prédictions et les reconvertit en texte si nécessaire.",
 			explain: " Le modèle ne peut pas comprendre le texte ! Le <i>tokenizer</i> doit d'abord tokeniser le texte et le convertir en identifiants afin qu'il soit compréhensible par le modèle."},
 		{
-			text: " Tout d'abord, le <i>tokenizer/i>, qui traite le texte et renvoie des identifiants. Puis le modèle traite ces identifiants et produit une prédiction, qui peut être du texte.",
+			text: " Tout d'abord, le <i>tokenizer</i>, qui traite le texte et renvoie des identifiants. Puis le modèle traite ces identifiants et produit une prédiction, qui peut être du texte.",
```
Comment (lewtun) [inline chapters/fr/chapter2/8.mdx:16]: Nice catch!

File: chapters/fr/_toctree.yml [hunk 2]
```diff
@@ -42,3 +42,75 @@
   - local: chapter5/8
     title: Quiz de fin de chapitre
     quiz: 5
+
+- title: 6. La bibliothèque 🤗 Tokenizer
+  sections:
+  - local: chapitre6/1
```
Comment (lewtun) [inline chapters/fr/_toctree.yml:48]: Ah sorry the instructions weren't super clear: we need to keep the filenames in English due to the way our frontend works. So all these `local` fields should be in English with e.g. `chapter6/1` here and so on

**Review by lewtun — COMMENTED** [review]
Wow, this is an incredible PR @lbourdois! Huge kudos on translating so many chapters in one go 🤗 !!

We just need to make some minor tweaks to the `local` fields in the `_toctree.yml` file, but otherwise this looks great

File: chapters/fr/_toctree.yml (hunk 2 above)
Comment (lbourdois) [reply chapters/fr/_toctree.yml]: It is modified :)

Comment (lbourdois) [reply chapters/fr/_toctree.yml]: Erf it did not allow to pass the test :'(

Comment (lewtun) [reply chapters/fr/_toctree.yml]: Thanks for the fix!

**Pull request merged**